backend/src/main/java/se/chalmers/dat265/group1/storage/repository/GenericRepository.java

```diff
@@ -143,20 +144,23 @@ public T updateObject(T object) {
 
     private T execute(JsonObject jsonObject2, StringBuilder query2) {
         try {
-            PreparedStatement ps = getPreparedStatementFromJSON(jsonObject2, query2); //maps model data to the query
+            PreparedStatement ps = getPreparedStatementFromJSON(jsonObject2, query2); //maps se.chalmers.dat265.group1.model data to the query
             ps.executeUpdate();
             ResultSet rs = ps.getGeneratedKeys();
             rs.next();
             int id = rs.getInt(1);
             return this.getObject(id);
         } catch (SQLException e) {
-            e.printStackTrace();
+            log.error("execute", e);
+            return null;
+        } finally {
+            System.out.println("Failed to execute");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)